randovania/gui/ui_files/preset_patcher_energy.ui

```diff
@@ -290,6 +283,59 @@
           </layout>
          </widget>
         </item>
+        <item>
+         <widget class="QGroupBox" name="damage_reduction_box">
+          <property name="title">
+           <string>Damage reduction</string>
+          </property>
+          <layout class="QGridLayout" name="gridLayout_5">
+           <item row="0" column="0">
+            <widget class="QLabel" name="damage_reduction_label">
+             <property name="text">
+              <string>
+                  &lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Configure
+                  how damage reduction is applied:&lt;/p&gt;&lt;p&gt;&lt;span
+                  style=&quot;
+                  font-weight:700;&quot;&gt;Default:
+                  &lt;/span&gt;Damage reduction is determined by the best suit you
+                  have (Varia - 10% reduction, Gravity - 20%, Phazon - 50%).&lt;/p&gt;&lt;p&gt;&lt;span
+                  style=&quot;
+                  font-weight:700;&quot;&gt;Progressive:
+                  &lt;/span&gt;Damage reduction is determined by how many suits you
+                  have (one suit - 10% reduction, two suits - 20%, all suits - 50%).&lt;/p&gt;&lt;p&gt;&lt;span
+                  style=&quot;
+                  font-weight:700;&quot;&gt;Additive:
+                  &lt;/span&gt;Each suit adds a specific
+                  amount of damage
+                  reduction (Varia/Gravity - +10% reduction each, Phazon - +30% reduction).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Are you using [Qt Creator](https://www.qt.io/product/development-tools) to edit this file? You can right click to change the text type to "rich" which enables you to easily add newlines, bullet points, etc.

![image](https://github.com/randovania/randovania/assets/14116556/707e8181-52dc-40cb-87c9-6c4061850fd8)

Please update this UI so it's not just one single line break per option

Reply:
Can you clarify and/or elaborate on what you meant? Do you just want me to fix the source code? Or did you mean you want to split the suit information into bullet points?

Reply:
![Screenshot_20240523_195807](https://github.com/randovania/randovania/assets/212321/333676df-da48-4e48-a77f-9f3013135c0b)

This is with bullet points and I'm not a fan of it.

Reply:
I think bullet points are preferable but I'm not necessarily going to die on that hill. The extra whitespace above the first bullet point is weird. See how other tabs handle that if that whitespace is what's making you "not a fan"

Reply:
![Screenshot_20240523_203855](https://github.com/randovania/randovania/assets/212321/57b8c883-9589-418d-ba2e-fe8f1207be2a)
This conveys the same information but more concisely. Thoughts?

Reply:
![Screenshot_20240523_204101](https://github.com/randovania/randovania/assets/212321/9cd5dea2-896c-4a67-a407-1b2e646b6ff4)
Added a clarification on Default.

Reply:
I prefer the variant with bullet points, particularly because Progressive is directly explained.

Reply:
fwiw i also prefer bullet points. getting rid of the space before the first point would be nice, but not the end of the world for me.

Reply:
I would also rephrase additive slightly to:
```
Varia - adds 10% reduction
Gravity - adds 10% reduction
Phazon - adds 30% reduction
```